pkg/virt-handler/vm.go

```diff
@@ -2308,23 +2309,27 @@ func (d *VirtualMachineController) getLauncherClientInfo(vmi *v1.VirtualMachineI
 	return launcherInfo
 }
 
-func (d *VirtualMachineController) vmUpdateHelperMigrationSource(origVMI *v1.VirtualMachineInstance) error {
-	client, err := d.getLauncherClient(origVMI)
-	if err != nil {
-		return fmt.Errorf(unableCreateVirtLauncherConnectionFmt, err)
+func isMigrationInProgress(vmi *v1.VirtualMachineInstance, domain *api.Domain) bool {
+	var domainMigrationMetadata *api.MigrationMetadata
+
+	if domain != nil {
+		domainMigrationMetadata = domain.Spec.Metadata.KubeVirt.Migration
+	}
+	if vmi.Status.MigrationState.StartTimestamp != nil ||
+		(domainMigrationMetadata != nil && domainMigrationMetadata.StartTimestamp != nil) {
```

Comment:
@acardace , I think this is not going to work. After the first migration, the domain metadata will have migration metadata. We can check here that the UID != vmi.Status.MigrationState.MigrationUID and that the EndTimestamp doesn't exist

Reply:
Fixed it, thank you!